packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -41,6 +41,8 @@ contract StableToken is
 
   event TransferComment(string comment);
 
+  bytes32 constant GRANDA_MENTO_REGISTRY_ID = keccak256(abi.encodePacked("GrandaMento"));
```

Comment:
Is it possible to do `registry.getAddressForStringOrDie` here?

Reply:
View calls like that aren't allowed outside of the constructor or functions. Also because the address may change in the registry it's best to use the registry ID